ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -1004,10 +1008,18 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
       return <FadeBox  style={{margin: 'auto', marginTop: '1rem', width: '95.7%'}}>
         <div style={{width: '1120px'}}>
           {loading && <SpinnerOverlay overrideStylesOverlay={styles.spinner}/>}
+          {!hasDefaultCdrVersion(this.state.workspace, cdrVersionListResponse) && showCdrVersionModal &&
+            <OldCdrVersionModal
+                onCancel={() => {
+                  this.setState(fp.set(['workspace', 'cdrVersionId'], cdrVersionListResponse.defaultCdrVersionId));
```

Comment:
Modal cancel: force the use of the default CDR
Modal continue (requires checkboxes): allows use of chosen older CDR